.gitignore

```diff
@@ -13,3 +13,4 @@
 
 node_modules/
 closure-deps/node_modules/
+package-lock.json
```

Comment:
It seems that lock files were not intentionally added, so I ignored.

Reply:
I'll think about adding this in the future, for this and the library.